drivers/wifi/nrf700x/osal/fw_if/umac_if/src/fmac_api.c

```diff
@@ -2701,6 +2701,41 @@ enum wifi_nrf_status wifi_nrf_fmac_get_station(void *dev_ctx,
 	return status;
 }
 
+enum wifi_nrf_status wifi_nrf_fmac_get_interface(void *dev_ctx,
+					       unsigned int if_idx)
+{
+	enum wifi_nrf_status status = WIFI_NRF_STATUS_FAIL;
+	struct img_cmd_get_interface *cmd = NULL;
+	struct wifi_nrf_fmac_dev_ctx *fmac_dev_ctx = NULL;
+
```

Comment:
Add defensive check to ensure that dev_ctx is not NULL and also to check that if_idx is less than MAX_NUM_VIFS

Reply:
Done